app/controllers/concerns/recaptcha_concern.rb

```diff
@@ -11,6 +11,14 @@ module RecaptchaConcern
     'https://recaptcha.google.com/recaptcha/',
   ].freeze
 
+  def add_recaptcha_resource_hints
+    response.headers['Link'] = [
+      response.headers['Link'],
+      '<https://www.google.com>;rel=preconnect',
+      '<https://www.gstatic.com>;rel=preconnect;crossorigin',
```

Comment:
One thing I didn't check previously is whether it's valid to omit the quotes on the values here, since both Rails and [related MDN resources](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link) include them.

I confirmed this in the relevant spec: https://httpwg.org/specs/rfc8288.html#header

>The ABNF for the field value is:
>
>```
>  Link       = #link-value
>  link-value = "<" URI-Reference ">" *( OWS ";" OWS link-param )
>  link-param = token BWS [ "=" BWS ( token / quoted-string ) ]
>```
>Note that any link-param can be generated with values using either the token or the quoted-string syntax; therefore, recipients MUST be able to parse both forms. In other words, the following parameters are equivalent:
>
>```
>  x=y
>  x="y"
>```

Adding on to my comment at https://github.com/18F/identity-idp/pull/10612#issuecomment-2108667459, this could be another micro-optimization if we wanted to fully control the resource hint header and optimize its size.

Reply:
Also validated them as working in Safari ([related blog post](https://andydavies.me/blog/2019/04/17/three-ways-of-checking-your-rel-equals-preconnect-resource-hints-are-working/)):

![image](https://github.com/18F/identity-idp/assets/1779930/7c21012c-ad12-4f94-980e-8310ab993a1c)